library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+    * Diffuse-slowing-hyperventilation <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [Diffuse slowing induced by hyperventilation. Bilateral, diffuse slowing during hyperventilation. Recorded in 70 percent of normal children (3-5 years) and less then 10 percent of adults. Usually appear in the posterior regions and spread forward in younger age group, whereas they tend to appear in the frontal regions and spread backward in the older age group. (Source: Beniczky ea 2013, Appendix S6; Beniczky ea 2017, Table 14.)]</nowiki>
+    * Photic-driving <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [Physiologic response consisting of rhythmic activity elicited over the posterior regions of the head by repetitive photic stimulation at frequencies of about 5-30 Hz. Comments: term should be limited to activity time-locked to the stimulus and of frequency identical or harmonically related to the stimulus frequency. Photic driving should be distinguished from the visual evoked potentials elicited by isolated flashes of light or flashes repeated at very low frequency. (Source: Beniczky ea 2013; Appendix S6, Beniczky ea 2017, Table 14.)]</nowiki>
+    * Photomyogenic-response <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [A response to intermittent photic stimulation characterized by the appearance in the record of brief, repetitive muscular artifacts (spikes) over the anterior regions of the head. These often increase gradually in amplitude as stimuli are continued and cease promptly when the stimulus is withdrawn. Comment: this response is frequently associated with flutter of the eyelids and vertical oscillations of the eyeballs and sometimes with discrete jerking mostly involving the musculature of the face and head. (Preferred to synonym: photo-myoclonic response). (Source: Beniczky ea 2013; Appendix S6, Beniczky ea 2017, Table 14.)]</nowiki>
+    * Arousal-pattern <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [Arousal pattern in children. Prolonged, marked high voltage 4-6/s activity in all leads with some intermixed slower frequencies, in children. (Source: Beniczky ea 2013, Appendix S6; Beniczky ea 2017, Table 14.)]</nowiki>
```

Comment:
In "intermixed slower frequencies, in children. "  the "in children" is redundant.

Reply:
fixed